services/openshiftbuilddeploymonitor/src/index.js

```diff
@@ -187,10 +190,21 @@ const messageConsumer = async msg => {
       }
 
       const routes = await getAllRoutesURLs()
-
       sendToLagoonLogs('info', projectName, "", `task:builddeploy-openshift:${buildPhase}`, meta,
         `*[${projectName}]* ${logMessage} Build \`${buildName}\` complete. ${logLink} \n ${routes.join("\n")}`
       )
+      try {
+        const updateEnvironmentResult = await updateEnvironment(
+          branchName,
+          environmentOpenshift.id,
+          `{
+            lagoon_route: "${routes[0]}",
```

Comment:
I don't think this will be always the Route that is saved within the ConfigMap as `LAGOON_ROUTE`. This just takes the first route that is found within the Routes object of Kubernetes.
I think we should actually load the configmap and the entries in `LAGOON_ROUTE` and `LAGOON_ROUTES` to fill the data in the API